include/zephyr/sys/mpsc_lockfree.h

```diff
@@ -97,10 +96,10 @@ struct rtio_mpsc {
  *
  * @param symbol name of the queue
  */
-#define RTIO_MPSC_INIT(symbol)                                                                     \
+#define MPSC_INIT(symbol)                                                                          \
 	{                                                                                          \
-		.head = (struct rtio_mpsc_node *)&symbol.stub,                                     \
-		.tail = (struct rtio_mpsc_node *)&symbol.stub,                                     \
+		.head = (struct mpsc_node *)&symbol.stub,                                          \
+		.tail = (struct mpsc_node *)&symbol.stub,                                          \
 		.stub = {                                                                          \
 			.next = NULL,                                                              \
 		},                                                                                 \
```

Comment:
In case consistency with SPSC and other few Zephyr APIs was aimed, `MPSC_INITIALIZER()` is also possible

https://github.com/zephyrproject-rtos/zephyr/pull/63470/files#diff-690e89323e13a9e22d9b6744504481620011234df581bbab923629217e0c529dR82

But it was already like that in RTIO, and seems like mostly for internal use anyway.

Reply:
May I ask why there is no `MPSC_DEFINE()`?

The use-case might have been limited to these infocation, and never be defined alone outside of RTIO context. Now someone might want to define a global MPSC queue.

https://github.com/zephyrproject-rtos/zephyr/pull/63470/files#diff-a0e010388d2114551593a77dfc3d6a51c2714c375411f3206af13cf6fee084f2R800-R801

Though, not hard to call `struct mpsc my_mpsc_queue = MSPC_INIT(my_mpsc_queue);` over `MPSC_DEFINE(my_mpsc_queue);`. Just one fewer repetition of the symbol name...